### PR TITLE
Merge 382 namespacing with master 

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- This performs a merge with master and adds it back to the `382-namespacing` branch,  essentially bringing it up to date with all the amazing work accomplished thus far.
- It removes some of the variable name changes, but I'll add them back in next.

# Needed By (Date)
- Soon. 

# Urgency
- We want to get this done before we get any more merge conflicts! 😆 

# Steps to Test

1. Check out this branch
1. Verify the code looks okay.
2. Verify Decanter still works- in particular the main nav stuff

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- JIRA ticket - https://stanfordits.atlassian.net/browse/THEME-14
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: core/src/scss/components/main-nav/_main-nav--dark.scss [hunk 1]
```diff
@@ -87,31 +85,31 @@
         &,
         &.su-main-nav__item--expanded {
           > a::after {
-            background: url("#{$su-image-path}/caret-down-white.svg") no-repeat 0 0;
+            background: url("#{$image-path}/caret-down-white.svg") no-repeat 0 0;
```
Comment (JBCSU) [inline core/src/scss/components/main-nav/_main-nav--dark.scss]:
```suggestion
            background: url("#{$su-image-path}/caret-down-white.svg") no-repeat 0 0;
```
I think this one needs to keep the `su-` prefix.

**Comment by cjwest** [comment]
Hi @JBCSU,
I added back in the variable name changes from `$image-path ` to `$su-image-path`
Let me know if there is anything else!
Otherwise please squash and merge.

File: core/src/scss/components/main-nav/_main-nav--dark.scss [hunk 2]
```diff
@@ -1,44 +1,42 @@
...
       &:active {
         &::before {
-          background-color: color(border--active, $_su-main-nav-color-map);
+          background-color: color(h-border--active, $nav-colors--dark);
         }
       }
 
       &::after {
-        background: url("#{$su-image-path}/hamburger-white.svg") no-repeat 3px 0;
+        background: url("#{$image-path}/hamburger-white.svg") no-repeat 3px 0;
```
Comment (JBCSU) [inline core/src/scss/components/main-nav/_main-nav--dark.scss]:
```suggestion
        background: url("#{$su-image-path}/hamburger-white.svg") no-repeat 3px 0;
```

File: core/src/scss/components/main-nav/_main-nav--dark.scss [hunk 3]
```diff
@@ -1,44 +1,42 @@
...
         }
 
         &:active::before {
-          background-color: color(border--expanded-active, $_su-main-nav-color-map);
+          background-color: color(h-border--expanded-active, $nav-colors--dark);
         }
 
         &::after {
-          background: url("#{$su-image-path}/close-white.svg") no-repeat 3px 0;
+          background: url("#{$image-path}/close-white.svg") no-repeat 3px 0;
```
Comment (JBCSU) [inline core/src/scss/components/main-nav/_main-nav--dark.scss]:
```suggestion
          background: url("#{$su-image-path}/close-white.svg") no-repeat 3px 0;
```

File: core/src/scss/utilities/placeholders/components/nav/_nav-item--parent-expanded.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+//
+// Individual Nav Item -- Parent Expanded
+//
+// Style guide: Placeholders.Components.Nav-Item-Parent-Expanded
+//
+
+%nav-item--parent-expanded {
+  > a {
+    &::after {
+      background: url("#{$image-path}/minus-white.svg") no-repeat 0 0;
```
Comment (JBCSU) [inline core/src/scss/utilities/placeholders/components/nav/_nav-item--parent-expanded.scss]:
```suggestion
      background: url("#{$su-image-path}/minus-white.svg") no-repeat 0 0;
```

**Comment by cjwest** [comment]
@JBCSU,

Good catch. I didn't get the changes saved. That's one thing I don't like about using the Atom IDE. When you save, it only saves the current file. I forgot to save _all_ the files.

File: core/src/scss/utilities/variables/components/_nav.scss [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+//    h-border--hover:                $color-bright-red,
+//    h-border--active:               $color-black,
+//    h-border--active-hover:         $color-bright-red,
+//    h-border--expanded:             $color-black,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.default
+//
+
+$nav-colors: (
```
Comment (sherakama) [inline core/src/scss/utilities/variables/components/_nav.scss]: Please be aware that these variables, and the rest of the nav, are being refactored currently in: https://github.com/SU-SWS/decanter/pull/443/files#diff-f94ee2cd63a360f0412ca184c56b3af9

Continue with what you are doing here but they will likely change in v6.

**Pull request merged**